components/Base/Tooltip/index.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <div v-if="isDropdown" id="tooltip" class="tooltip">
+    <div class="tooltip__content">
+      <div class="tooltip__content-bar">
+        <div
+          :class="{
+            'tooltip__content-bar--default' : true,
+            'tooltip__content-bar--background-low' : level === 'low',
+            'tooltip__content-bar--background-medium' : level === 'medium',
+            'tooltip__content-bar--background-strong' : level === 'strong'
+          }"
+          v-bind="{...$props,...$attrs}"
+        />
+        <div
+          :class="{
+            'tooltip__content-bar--default' : true,
+            'tooltip__content-bar--background-medium' : level === 'medium',
+            'tooltip__content-bar--background-strong' : level === 'strong'
+          }"
+          v-bind="{...$props,...$attrs}"
+        />
+        <div
+          :class="{
+            'tooltip__content-bar--default' : true,
+            'tooltip__content-bar--background-strong' : level === 'strong'
+          }"
+          v-bind="{...$props,...$attrs}"
+        />
+      </div>
+      <div class="tooltip__content-text">
+        Kata Sandi Anda
+        <span
+          v-if="isLevel"
+          :class="{
+            'tooltip__content-text--weight' : true,
+            'tooltip__content-text--color-low' : level === 'low',
+            'tooltip__content-text--color-medium' : level === 'medium',
+            'tooltip__content-text--color-strong' : level === 'strong',
+          }"
+          v-bind="{...$props,...$attrs}"
+        >
+          {{ textName }}
+        </span>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  inheritAttrs: false,
+  props: {
+    level: {
+      type: String,
+      default: 'default'
+    },
+    isDropdown: {
+      type: Boolean,
+      default: false
+    }
+  },
+  data () {
+    return {
+      textName: '',
+      options: [
+        {
+          keys: 'low',
+          text: 'tidak cukup kuat.'
+        },
+        {
+          keys: 'medium',
+          text: 'cukup kuat.'
+        },
+        {
+          keys: 'strong',
+          text: 'kuat.'
+        }
+      ]
+    }
+  },
+  computed: {
+    isLevel () {
+      if (this.level !== null) {
+        return true
+      } else {
+        return false
+      }
```

Comment:
**minor**:
maybe this one is enough #cmiiw 
```
isLevel () {
  return this.level !== null
}
```

Reply:
Yes, I think it is the same as I wrote and readable 👍🏽